lib/galaxy/managers/cloud.py

```diff
@@ -203,7 +203,7 @@ def upload(self, trans, history_id, provider, bucket, objects, credentials, inpu
 
         connection = self._configure_provider(provider, credentials)
         try:
-            bucket_obj = connection.object_store.get(bucket)
+            bucket_obj = connection.storage.buckets.get(bucket)
```

Comment:
Just a minor nit, but it might be better to call this bucket instead of bucket_obj, just so it doesn't cause any confusion with BucketObject.